doc/engineering-reference/src/simulation-models-encyclopedic-reference-005/occupant-thermal-comfort.tex

```diff
@@ -227,15 +227,15 @@ \subsubsection{Description of the model and algorithm}\label{description-of-the-
 \end{equation}
 
 \begin{equation}
-{Q_{r}}=3.96*f\left(_{l}\right)*{[}(0.01*T\left(_{a}\right))\left(^{4}\right)-(0.01*T\left(_{a}\right))\left(^{4}\right){]}
+{Q_{r}}=3.96*f_{cl}*{[}(0.01*T_{cla})^{4}-(0.01*T_{ra})^{4}{]}
```

Comment:
Interesting, OK.  Definitely cleaner, but I think it's changing the equation, right?  I'll have to peek at before/after.